src/index.js

```diff
@@ -88,18 +92,41 @@ function visitEntity(entity, entitySchema, bag, options) {
   let normalized = visitObject(entity, entitySchema, bag, options);
   mergeIntoEntity(stored, normalized, entityKey);
 
+  if(entitySchema._mappedBy) {
+    const fk = entitySchema._mappedBy;
+    if(entitySchema[fk] instanceof IterableSchema && parentId) {
+      if(!stored[fk]) {
+        stored[fk] = [];
+      }
+      stored[fk] = Array.from(new Set([
```

Comment:
Same for https://developer.mozilla.org/en/docs/Web/JavaScript/Reference/Global_Objects/Array/from:

<img width="481" alt="screen shot 2016-04-07 at 16 14 13" src="https://cloud.githubusercontent.com/assets/810438/14356192/cab5b320-fcdb-11e5-9e59-c99de9c154fe.png">